
  0%|          | 0/705 [00:00<?, ?it/s]
Error in callback <bound method _WandbInit._pause_backend of <wandb.sdk.wandb_init._WandbInit object at 0x00000243E5BA91E0>> (for post_run_cell), with arguments args (<ExecutionResult object at 243633bdb70, execution_count=20 error_before_exec=None error_in_exec=list index out of range info=<ExecutionInfo object at 243633bd6c0, raw_cell="from transformers import Trainer, TrainingArgument.." store_history=True silent=False shell_futures=True cell_id=vscode-notebook-cell:/c%3A/Users/60130060/hse-nlp/homeworks/hw4/hw4.ipynb#X45sZmlsZQ%3D%3D> result=None>,),kwargs {}:
Error in callback <bound method _WandbInit._resume_backend of <wandb.sdk.wandb_init._WandbInit object at 0x00000243E5BA91E0>> (for pre_run_cell), with arguments args (<ExecutionInfo object at 243f0491690, raw_cell="import builtins
import ipykernel
builtins.print(ip.." store_history=False silent=False shell_futures=True cell_id=None>,),kwargs {}:
6.27.1
Error in callback <bound method _WandbInit._pause_backend of <wandb.sdk.wandb_init._WandbInit object at 0x00000243E5BA91E0>> (for post_run_cell), with arguments args (<ExecutionResult object at 243f0491450, execution_count=None error_before_exec=None error_in_exec=None info=<ExecutionInfo object at 243f0491690, raw_cell="import builtins
import ipykernel
builtins.print(ip.." store_history=False silent=False shell_futures=True cell_id=None> result=None>,),kwargs {}:
Error in callback <bound method _WandbInit._resume_backend of <wandb.sdk.wandb_init._WandbInit object at 0x00000243E5BA91E0>> (for pre_run_cell), with arguments args (<ExecutionInfo object at 243633beb90, raw_cell="class Dataset:
    def __init__(self, raw_dataset:.." store_history=True silent=False shell_futures=True cell_id=vscode-notebook-cell:/c%3A/Users/60130060/hse-nlp/homeworks/hw4/hw4.ipynb#X21sZmlsZQ%3D%3D>,),kwargs {}:
Error in callback <bound method _WandbInit._pause_backend of <wandb.sdk.wandb_init._WandbInit object at 0x00000243E5BA91E0>> (for post_run_cell), with arguments args (<ExecutionResult object at 243633bc460, execution_count=21 error_before_exec=None error_in_exec=None info=<ExecutionInfo object at 243633beb90, raw_cell="class Dataset:
    def __init__(self, raw_dataset:.." store_history=True silent=False shell_futures=True cell_id=vscode-notebook-cell:/c%3A/Users/60130060/hse-nlp/homeworks/hw4/hw4.ipynb#X21sZmlsZQ%3D%3D> result=None>,),kwargs {}:
Error in callback <bound method _WandbInit._resume_backend of <wandb.sdk.wandb_init._WandbInit object at 0x00000243E5BA91E0>> (for pre_run_cell), with arguments args (<ExecutionInfo object at 243f0490f40, raw_cell="data_collator = DataCollatorForTokenClassification.." store_history=True silent=False shell_futures=True cell_id=vscode-notebook-cell:/c%3A/Users/60130060/hse-nlp/homeworks/hw4/hw4.ipynb#X25sZmlsZQ%3D%3D>,),kwargs {}:
Error in callback <bound method _WandbInit._pause_backend of <wandb.sdk.wandb_init._WandbInit object at 0x00000243E5BA91E0>> (for post_run_cell), with arguments args (<ExecutionResult object at 243f0491270, execution_count=22 error_before_exec=None error_in_exec=None info=<ExecutionInfo object at 243f0490f40, raw_cell="data_collator = DataCollatorForTokenClassification.." store_history=True silent=False shell_futures=True cell_id=vscode-notebook-cell:/c%3A/Users/60130060/hse-nlp/homeworks/hw4/hw4.ipynb#X25sZmlsZQ%3D%3D> result=None>,),kwargs {}:
Error in callback <bound method _WandbInit._resume_backend of <wandb.sdk.wandb_init._WandbInit object at 0x00000243E5BA91E0>> (for pre_run_cell), with arguments args (<ExecutionInfo object at 243f0491150, raw_cell="batch_size = 64
train_loader = DataLoader(
    tr.." store_history=True silent=False shell_futures=True cell_id=vscode-notebook-cell:/c%3A/Users/60130060/hse-nlp/homeworks/hw4/hw4.ipynb#X31sZmlsZQ%3D%3D>,),kwargs {}:
Error in callback <bound method _WandbInit._pause_backend of <wandb.sdk.wandb_init._WandbInit object at 0x00000243E5BA91E0>> (for post_run_cell), with arguments args (<ExecutionResult object at 243f0490640, execution_count=23 error_before_exec=None error_in_exec=None info=<ExecutionInfo object at 243f0491150, raw_cell="batch_size = 64
train_loader = DataLoader(
    tr.." store_history=True silent=False shell_futures=True cell_id=vscode-notebook-cell:/c%3A/Users/60130060/hse-nlp/homeworks/hw4/hw4.ipynb#X31sZmlsZQ%3D%3D> result=None>,),kwargs {}:
Error in callback <bound method _WandbInit._resume_backend of <wandb.sdk.wandb_init._WandbInit object at 0x00000243E5BA91E0>> (for pre_run_cell), with arguments args (<ExecutionInfo object at 243f04919f0, raw_cell="#! pip install seqeval" store_history=True silent=False shell_futures=True cell_id=vscode-notebook-cell:/c%3A/Users/60130060/hse-nlp/homeworks/hw4/hw4.ipynb#X33sZmlsZQ%3D%3D>,),kwargs {}:
Error in callback <bound method _WandbInit._pause_backend of <wandb.sdk.wandb_init._WandbInit object at 0x00000243E5BA91E0>> (for post_run_cell), with arguments args (<ExecutionResult object at 243f04910f0, execution_count=24 error_before_exec=None error_in_exec=None info=<ExecutionInfo object at 243f04919f0, raw_cell="#! pip install seqeval" store_history=True silent=False shell_futures=True cell_id=vscode-notebook-cell:/c%3A/Users/60130060/hse-nlp/homeworks/hw4/hw4.ipynb#X33sZmlsZQ%3D%3D> result=None>,),kwargs {}:
  0%|          | 0/705 [00:00<?, ?it/s]Some weights of BertForTokenClassification were not initialized from the model checkpoint at bert-base-cased and are newly initialized: ['classifier.weight', 'classifier.bias']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
Number of parameters: 107726601
